ext/pg_result.c

```diff
@@ -527,6 +527,32 @@ pgresult_error_message(VALUE self)
 	return ret;
 }
 
+#ifdef HAVE_PQRESULTVERBOSEERRORMESSAGE
```

Comment:
We should mention the PostgreSQL version in rdoc when we conditionally define methods. It looks like our convention is a single line at the end of the comment: `Available since PostgreSQL-9.6`

https://github.com/ged/ruby-pg/blob/8c1b2a44abb2a137a7fe78816f7386fac504fd5f/ext/pg_connection.c#L3373-L3381